### PR TITLE
Migrating WT3 cooling tower model

### PR DESCRIPTION
**Pull request by luohezhiming**
## Summary/Motivation:

Add cooling tower model

## Changes proposed in this PR:
- add cooling tower model
- add corresponding tests

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/unit_models/zero_order/cooling_tower_zo.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
```
Comment (tarnold17) [inline watertap/unit_models/zero_order/cooling_tower_zo.py:2]: Should the copyright date in the header still be 2021 even though it is now 2022?

Comment (luohezhiming) [reply watertap/unit_models/zero_order/cooling_tower_zo.py:2]: I didn't pay attention to this before, but all our migrated models from WT3 use the same copyright. @adam-a-a

Comment (andrewlee94) [reply watertap/unit_models/zero_order/cooling_tower_zo.py:2]: We should not change the copyright header ourselves. Firstly, there is a lot of paperwork that goes into getting it approved, and secondly there is an automated tool for updating the headers when required.

File: watertap/data/techno_economic/cooling_tower.yaml [hunk 2]
```diff
@@ -1,8 +1,30 @@
-default:
+default: # make_up_mass_balance method in the WT3 cooling_tower.py is used
+  energy_electric_flow_vol_inlet:
+    value: 0.000000001
```
Comment (andrewlee94) [inline watertap/data/techno_economic/cooling_tower.yaml]: I just want to confirm this value before proceeding - it is very small. @kurbansitterley Are the units correct, or is this value meant to be effectively zero (but given a small positive value instead)?

**Review by andrewlee94 — DISMISSED** [review]
Looks good, but I had a question about the value of electricity intensity.

File: watertap/data/techno_economic/cooling_tower.yaml (hunk 2 above)
Comment (kurbansitterley) [reply watertap/data/techno_economic/cooling_tower.yaml]: Should be effectively zero.

**Review by kurbansitterley — DISMISSED** [review]

**Review by andrewlee94 — APPROVED** [review]

**Review by adam-a-a — APPROVED** [review]

**Pull request merged**